src/reanimated2/layoutReanimation/web/animationsManager.ts

```diff
@@ -94,6 +94,10 @@ export function startWebLayoutAnimation<
   shouldMakeVisible = false,
   transitionData?: TransitionData
 ) {
+  if (shouldMakeVisible) {
+    makeElementVisible(element);
+  }
+
```

Comment:
It looks like now we ignore delay (look [here](https://github.com/software-mansion/react-native-reanimated/blob/4aac80fbd90f76a778bd10e42576dbb0822e0699/src/reanimated2/layoutReanimation/web/componentUtils.ts#L162) for context)

Reply:
How do we ignore it? From what I see previous 'unhides' were made without the delay.

Reply:
Yes, they were, but only because we didn't trigger any animation after revealing this element. If I understand correctly, now if element has `entering` prop `makeElementVisible` will trigger, which means that we are ignoring delay that I mentioned in link above.

Reply:
I read it very closely and you're absolutely right - but I just hate it that we have different control flows and every one of them calls this function. I will give it a big think later on and come up with a universal solution.